ivy/functional/frontends/torch/reduction_ops.py

```diff
@@ -251,3 +251,11 @@ def unique(input, sorted=True, return_inverse=False, return_counts=False, dim=No
         values.append(results.counts)
 
     return Results(*values)
+
+
+@to_ivy_arrays_and_back
+def mode(input, dim=-1, keepdim=False, *, out=None):
+    unique_data, counts = ivy.unique(input, return_counts=True, dim=dim)

```

Comment:
There is no `unique` function in our api. We've got a `unique_all` function which you can read about in our docs. This is causing an error which you should be able to see if you run the test locally.

Reply:
Sorry, I was taking help from the torch doc.